internal/csi-addons/rbd/reclaimspace.go

```diff
@@ -90,12 +98,13 @@ func (rscs *ReclaimSpaceControllerServer) ControllerReclaimSpace(
 // of CSI-addons reclaimspace controller service spec.
 type ReclaimSpaceNodeServer struct {
 	*rs.UnimplementedReclaimSpaceNodeServer
+	*util.VolumeLocks
 }
 
 // NewReclaimSpaceNodeServer creates a new IdentityServer which handles the
 // Identity Service requests from the CSI-Addons specification.
 func NewReclaimSpaceNodeServer() *ReclaimSpaceNodeServer {
-	return &ReclaimSpaceNodeServer{}
+	return &ReclaimSpaceNodeServer{VolumeLocks: util.NewVolumeLocks()}
```

Comment:
we should reuse the lock from the node server like https://github.com/ceph/ceph-csi/pull/4690/files#diff-d44bddac57ff52af0445463bf6fe3e03b7aed04b3d41ea639d35c01522a74808R233. can you please address this one.

Reply:
Okay, so will share VolumeLocks? Then I'll close this https://github.com/ceph/ceph-csi/issues/4648?

Reply:
@iPraveenParihar yes please